python/paddle/__init__.py

```diff
@@ -35,6 +35,8 @@
 from .framework import disable_static  # noqa: F401
 from .framework import enable_static  # noqa: F401
 from .framework import in_dynamic_mode  # noqa: F401
+from .fluid.layers import l2_normalize  # noqa: F401
```

Comment:
shall we use `paddle.divide` and `paddle.linalg.norm` instead of l2_normalize, so no need import l2_normalize.

Reply:
done